mathOperations.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * 3. Basic Mathematical Operations
+ 
+ * DESCRIPTION:
+    Your task is to create a function that does four basic mathematical operations.
+    The function should take three arguments - operation(string/char), value1(number), value2(number).
+    The function should return result of numbers after applying the chosen operation.
+
+    Examples(Operator, value1, value2) --> output
+        ('+', 4, 7) --> 11
+        ('-', 15, 18) --> -3
+        ('*', 5, 5) --> 25
+        ('/', 49, 7) --> 7
+
+ */
+
+function mathOperations(operator, num1, num2) {
+    var res = 0;
+    switch (operator) {
+        case '+':
+            res = num1 + num2;
+            break;
+        case '-':
+            res = num1 - num2;
+            break;
+        case '*':
+            res = num1 * num2;
+            break;
+        case '/':
+            res = num1 / num2;
+            break;
+        default:
+            console.log("Specify a valid operator");
+            return;
+    }
+    console.log("(\'" + operator + "\', " + num1 + ", " + num2 + ") " + "--> " + res);
```

Comment:
suggestion: Move the console statement outside of the function. Return the result from the function

Reply:
Sure Ratheesh.